src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -1229,9 +1230,10 @@ export class TabsTitleControl extends TitleControl {
 		}
 
 		const tabSizing = isTabSticky && options.pinnedTabSizing === 'shrink' ? 'shrink' /* treat sticky shrink tabs as tabSizing: 'shrink' */ : options.tabSizing;
-		for (const option of ['fit', 'shrink']) {
+		for (const option of ['fit', 'shrink', 'fixed']) {
 			tabContainer.classList.toggle(`sizing-${option}`, tabSizing === option);
 		}
+		tabContainer.style.width = `${options.tabSizingFixedWidth}px`;
```

Comment:
This can only be applied conditionally based on the setting.

Reply:
d'oh!